docs/deep-dives/specifications/fml-spec.mdx

```diff
@@ -457,11 +477,11 @@ In this way, app developers can use the feature manifest as a replacement for ot
 
 Primitive types supported:
 
- * `String` or `string`
- * `Boolean`, `Bool` or `boolean`. Both Kotlin and Swift's spelling is supported.
```

Comment:
Will deprecating these aliases cause breakage to existing manifests?

Reply:
No, they were never supported :(